src/main/java/com/thealgorithms/maths/HeronsFormula.java

```diff
@@ -1,18 +1,27 @@
 package com.thealgorithms.maths;
 
 /**
+ * Wikipedia for HeronsFormula => https://en.wikipedia.org/wiki/Heron%27s_formula
  * Find the area of a triangle using only side lengths
  */
 
-public class HeronsFormula {
+public final class HeronsFormula {
 
-    public static double Herons(int s1, int s2, int s3) {
-        double a = s1;
-        double b = s2;
-        double c = s3;
-        double s = (a + b + c) / 2.0;
-        double area = 0;
-        area = Math.sqrt((s) * (s - a) * (s - b) * (s - c));
-        return area;
+    /*
+     * A function to get the Area of a Triangle using Heron's Formula
+     * @param s1,s2,s3 => the three sides of the Triangle
+     * @return area using the formula (√(s(s – s1)(s – s2)(s – s3)))
+     * here s is called semi-perimeter and it is the half of the perimeter (i.e; s = (s1+s2+s3)/2)
+     * @author satyabarghav
+     */
+    private HeronsFormula() {
+    }
+    public static double Herons(final double a, final double b, final double c) {
+        if (!(a + b > c && b + c > a && c + a > b)) {
```

Comment:
What do you think about changing the inequalities to `>=`? I think there is nothing wrong in a triangle with side lengths 1, 2, 3 (i.e. it has are 0). Moreover, I would suggest to add a check if the provided side lengths are `>= 0`.

```suggestion
        if (!(a + b >= c && b + c >= a && c + a >= b)) {
```

Reply:
I don't think that's correct according to the definition and a triangle with sides 1,2,3 cannot be constructed Geometrically
![image](https://github.com/TheAlgorithms/Java/assets/36763910/24c3caac-64fc-4599-8be1-488f11a19304)

Reply:
A _"1, 2, 3"_-triangle would be empty. I would not exclude it. But I understand that it might be a bit controversial. 

Please add a check that provided side lengths are all positive.